devel/ghidra/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           app 1.0
+
+name                ghidra
+homepage            https://ghidra-sre.org/
+master_sites        https://ghidra-sre.org/
```

Comment:
You can set `master_sites` to `homepage` since they are the same:
```
master_sites ${homepage}
```

Reply:
Good spot.  Fixed.